src/protocols/rdp/Makefile.am

```diff
@@ -241,7 +241,8 @@ rdp_keymaps =                                \
     $(srcdir)/keymaps/pt_br_qwerty.keymap    \
     $(srcdir)/keymaps/sv_se_qwerty.keymap    \
     $(srcdir)/keymaps/da_dk_qwerty.keymap    \
-    $(srcdir)/keymaps/tr_tr_qwerty.keymap
+    $(srcdir)/keymaps/tr_tr_qwerty.keymap    \
+    $(srcdir)/keymaps/no_no_qwerty.keymap
```

Comment:
With the exception of "base" and "failsafe" (which are special cases) and "da_dk_qwerty" (which somehow slipped in here unsorted), the files listed here are being maintained in sorted order. Please move your new keymap up in the list such that sorted order is maintained.